scripts/core/superdesk-api.d.ts

```diff
@@ -1362,6 +1372,7 @@ declare module 'superdesk-api' {
             formatDate(date: Date): string;
             formatDateTime(date: Date): string;
             longFormatDateTime(date: Date | string): string;
+            localeForDatePicker(targetLocale?: string): DatePickerLocaleSettings;
```

Comment:
It may be used for other things than date picker, for example to print dates in different components, so I'd make the naming more generic like `getDateTimePeriodNames`

Reply:
See my comment above as to the usage of this helper function